conans/client/client_cache.py

```diff
@@ -244,7 +245,7 @@ def load_manifest(self, conan_reference):
         """conan_id = sha(zip file)"""
         assert isinstance(conan_reference, ConanFileReference)
         export_folder = self.export(conan_reference)
-        check_ref_case(conan_reference, export_folder, self.store)
+        check_ref_case(conan_reference, self.store)
```

Comment:
check_ref_case checks the existence of the folder, not the store, to check for conflicts. I suggest leaving it as-is.

Reply:
The previous implementation was using that argument only in the first lines, I don't know how relevant this check was... and it skips the case-sensitiveness trial that is performed in the next lines:

```python

def check_ref_case(conan_reference, conan_folder, store_folder):	
        if not os.path.exists(conan_folder):  # If it doesn't exist, not a problem	
            return
        ...
        # Check case-sensitiveness issues
```

Even more, I don't know why this check was being done only for the `load_manifest` function, and the `conanfile` one.